app/views/content/funding-and-support/if-youre-a-veteran.md

```diff
@@ -31,11 +31,11 @@ There are full and part-time teacher training courses available.
 
 ## Undergraduate bursary for veterans
 
-You may be eligible for a tax-free undergraduate bursary of £40,000 if you’re (all of the following):
+You may be eligible for a tax-free undergraduate bursary of £40,000 if you (all of the following):
```

Comment:
```suggestion
You may be eligible for a tax-free undergraduate bursary of £40,000 if you are (all of the following):
```
Feels a bit funny like it doesn't read write the 'if you (all of the following)'. Could we amend slightly?